ui/raidboss/common_replacement.js

```diff
@@ -11,7 +11,7 @@ let commonReplacement = {
   },
   'is no longer sealed': {
     de: 'öffnet sich erneut',
-    fr: 'Ouverture ',
+    fr: 'Ouverture d(?:e|u|es) $1 dans',
```

Comment:
I'm not sure I understand why? I think this was fine before?

Reply:
I answered below